model/utils.go

```diff
@@ -573,3 +573,26 @@ func IsDomainName(s string) bool {
 
 	return ok
 }
+
+func RemoveDuplicateStrings(in []string) []string {
+	out := []string{}
+	seen := make(map[string]bool)
+
+	for _, item := range in {
+		if !seen[item] {
+			out = append(out, item)
+
+			seen[item] = true
+		}
+	}
+
+	return out
+}
+
+func GetPreferredTimezone(timezone StringMap) string {
```

Comment:
This duplicates code in `model/user.go`'s `User.GetPreferredTimezone`. Can we refactor that one to use this helper method?

Reply:
i guess done